Client/Frontend/Sync/SyncSettingsTableViewController.swift

```diff
@@ -107,8 +125,9 @@ class SyncSettingsTableViewController: UITableViewController {
             return
         }
         
-        guard let frc = frc, let deviceCount = frc.fetchedObjects?.count else { return }
-        let device = frc.object(at: indexPath)
+        let devices = self.devices
+        guard !devices.isEmpty else { return }
+        let device = devices[indexPath.row]
```

Comment:
for safety, you can use safe subscript here to make sure the device exist at given index path
`devices[safe: indexPath.row]`